### ISSUE TITLE
Investigate perplexity difference without answer spans in input text (only as segements)

### ISSUE THREAD

**Issue by martiansideofthemoon** [martiansideofthemoon/squash-generation#18]

**Comment by santhoshkolloju** [comment]
do u think retraining the model by removing the answer spans and just adding only the segmentation info will any impact on the performance?
<bos> .. paragraph text <question-general> .. question span .. <eos>`
token_types = [
            answer_general if ((i - 1) >= ans_start and (i - 1) < ans_end) else paragraph
            for i in range(len(curr_para) + 1)
        ]

**Comment by martiansideofthemoon** [comment]
I don't think it should impact performance, it should also make batching possible. The main issue is that finding the right alignment is a bit hard due to word piece mismatch

**Comment by santhoshkolloju** [comment]
Okay thanks for the response will try retraining in colab. As you said i think it will make batching easy. & 
Padding mismatch between training and inference will be eliminated.
